source/features/batch-open-conversations.tsx

```diff
@@ -63,12 +78,44 @@ async function init(): Promise<void | false> {
 				Open selected
 			</button>
 		);
+	} else {
+		select('#js-issues-toolbar')?.prepend(
```

Comment:
This is being selected twice. Also don’t use `?.` because it silenced runtime errors, which we need to see when features break. 

I think this needs to be behind a condition, we shouldn’t add this UI when there are no visible issues

Reply:
### `?.` 

i used the same like in line 60 🤔 
```shell
select('.table-list-header-toggle:not(.states)')?.prepend(
```

### selected twice

i make a variable for it

i guess you suggest the `!`? because if i check the variable for null then it is again a "silent error"

### behind a condition

there is an early exit in line 53 which show the batch function only if there are 2 issues

```javascript
async function init(): Promise<void | false> {
	if (!await elementReady('.js-issue-row + .js-issue-row')) {
		return false;
	}
```

Reply:
I think it’s good now. Thanks